volttron/platform/vip/zmq_connection.py

```diff
@@ -106,6 +106,10 @@ def register(self, handler):
     def send_vip_object(self, message, flags=0, copy=True, track=False):
         self.socket.send_vip_object(message, flags, copy, track)
 
+    def send_vip(self, peer, subsystem, args = None, msg_id = b'',
```

Comment:
please modify so that there are not spaces between the argument and the value....so args=None NOT args = None.

Reply:
Done